File: C++/1637. Widest Vertical Area Between Two Points Containing No Points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& p) {
        sort(p.begin(),p.end());
        int i,m=0;
        for(i=0;i<p.size()-1;i++){
            if(m<(p[i+1][0]-p[i][0]))
            m=p[i+1][0]-p[i][0];
        }
        return m;
    }
};

File: JAVA/1637. Widest Vertical Area Between Two Points Containing No Points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] p) {
        int[] a=new int[p.length];
        int i,m=0;
        for(i=0;i<p.length;i++){
            a[i]=p[i][0];
        }
        Arrays.sort(a);
        for(i=0;i<a.length-1;i++){
            if((a[i+1]-a[i])>m)
            m=a[i+1]-a[i];
        }
        return m;
    }
}